### PR TITLE
Skia : Fixes NullReferenceException for DefaultTypeface and adds .otf support

### PR DESCRIPTION
**Pull request by Gillibald**
- What does the pull request do?
It fixes an issue that causes crashes on some systems when our defined default typeface can't be resolved.  Now it should use the system's default in that case. It also adds .otf support.
- What is the current behavior?
The application crashes when the default can't be found.
- What is the updated/expected behavior with this PR?
The application should be able to resolve at least the system's default font.
- How was the solution implemented (if it's not obvious)?

### CODE REVIEW

File: src/Avalonia.Visuals/Media/Fonts/FamilyNameCollection.cs [hunk 1]
```diff
@@ -24,7 +24,10 @@ public FamilyNameCollection(IEnumerable<string> familyNames)
 
             var names = new List<string>(familyNames);
 
-            if (names.Count == 0) throw new ArgumentException($"{nameof(familyNames)} must not be empty.");
+            if (names.Count == 0)
```
Comment (Kryptos-FR) [inline src/Avalonia.Visuals/Media/Fonts/FamilyNameCollection.cs]: Automatically formatting the file might not be a good idea :)
Unrelated changes such as that one should be avoided IMO.

Comment (Gillibald) [reply src/Avalonia.Visuals/Media/Fonts/FamilyNameCollection.cs]: That is not automated that was my intention to fix the code style. I don't see why we shouldn't fix code style when we work on something. The code base is so big I don't think anyone ever puts its hands on this otherwise.
